src/eslint-config-adeira/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - Added `react/jsx-no-script-url` as next_version_error
+- `eslint-comments/no-unlimited-disable` rule is turned on, ignoring all `__generated__/*.graphql.js` files by default, so files autogenerated by Relay compiler are not reported. Follow [Eslint Docs](https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files) on how to disable additional group of files if needed.
```

Comment:
Can we add here the `overrides` example as well to make it easier?

Reply:
fixed!